src/main/java/com/deusgmbh/xcusatio/ui/XCusatioWindow.java

```diff
@@ -1,13 +1,153 @@
 package com.deusgmbh.xcusatio.ui;
 
+import java.util.ArrayList;
+import java.util.function.Consumer;
+
+import com.deusgmbh.xcusatio.data.scenarios.Scenario;
+import com.deusgmbh.xcusatio.ui.dashboard.Dashboard;
+import com.deusgmbh.xcusatio.ui.editor.Editor;
+import com.deusgmbh.xcusatio.ui.profilsettings.ProfileSettings;
+
 import javafx.application.Application;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.VBox;
 import javafx.stage.Stage;
 
+/**
+ * 
+ * This is the main class for the User Interface. This class creates the main
+ * scene and merges all components
+ * 
+ * @author Pascal.Schroeder@de.ibm.com
+ *
+ */
+
 public class XCusatioWindow extends Application {
+    private static final String WINDOW_TITLE = "Deus GmbH - xCusatio";
+    private static final int WINDOW_DEF_WIDTH = 800;
+    private static final int WINDOW_DEF_HEIGHT = 600;
+
+    private static final String NAVIGATION_PANEL_BACKGROUND_COLOR = "#888888";
+    private static final double NAVIGATION_PANEL_WIDTH_MULTIPLIER = 0.1;
+    private static final double NAVIGATION_PANEL_BUTTON_HEIGHT_MULTIPLIER = 0.1;
+
+    private static final String DASHBOARD_TAB_NAME = "Zur�ck";
+    private static final String EDITOR_TAB_NAME = "Editor";
+    private static final String PROFILE_SETTINGS_TAB_NAME = "Profile";
+
+    private BorderPane main;
+    private VBox navigationPanel;
+    private Dashboard dashboard;
+    private Editor editor;
+    private ProfileSettings profileSettings;
+
     @Override
     public void start(Stage stage) throws Exception {
+        initMainStage(stage);
+        dashboard = new Dashboard();
+        editor = new Editor();
+        profileSettings = new ProfileSettings();
+        initNavigationPanel();
+
+        main.setLeft(navigationPanel);
+        main.setCenter(dashboard);
 
         stage.show();
     }
 
+    public void createScenarioButtons(ArrayList<Scenario> scenarioList, Consumer<String> generateExcuse) {
```

Comment:
Diese Funktion gehört in die Dashboard Klasse

Reply:
done